source/rust_verify/src/config.rs

```diff
@@ -275,6 +277,11 @@ pub fn parse_args_with_imports(
         OPT_RECORD,
         "Rerun verus and package source files of the current crate to the current directory, alongside with output and version information. The file will be named YYYY-MM-DD-HH-MM-SS.zip. If you are reporting an error, please keep the original arguments in addition to this flag",
     );
+    opts.optflag(
+        "",
+        OPT_LOCAL_STORE,
+        "Record your changes to a specific crate (and its dependencies) everytime it invokes verus.",
+    );
```

Comment:
Rather than a command-line flag, we probably want it to be something project specific (with the possibility of the user opting in everywhere if they wish to). My suggestion is to check if there is a `.verus-opt-in` (as the full name of the file; this is not an extension, just the whole file name, including the `.`) in the same directory, or any ancestor directory, of the `foo.rs` file that was passed. If that file exists, then it should store to local store.